src/stream/layer/tracker/bytes.rs

```diff
@@ -400,6 +400,6 @@ mod tests {
         assert_eq!(handle.read(), 9);
         assert_eq!(handle.written(), 9);
 
-        futures::future::join_all(vec![task_1, task_2]).await;
+        futures_lite::future::zip(task_1, task_2).await;
```

Comment:
You can do

```
let (t1, t2) = futures_lite...
t1.unwrap();
t2.unwrap();
```

This might fail as it was previously ignored already as well. Do let me know in case that happens.